_news/107.md

```diff
@@ -23,9 +23,9 @@ Some highlights of the release are:
 
 You can find the Release Notes with more detailed info at
 
-<https://www.erlang.org/download/otp_src_19.1.readme>
```

Comment:
keep the `www`

Reply:
But it is redirecting to erlang.org

Reply:
But it is redirecting to 

> ```
> doc/search         /doc
> ```
> 
> Is actually wrong in the redirect and should be removed. I hadn't noticed it as netlify does not redirect any URLs that it can serve, and /doc/search does exist. So [5df2f80](https://github.com/erlang/erlang-org/commit/5df2f80b17f0868f4b50fcfcafe01996de4d4fe8) should not be done.

So should it discard the two commits that replace `/doc/search` ?

Reply:
> But it is redirecting to erlang.org

www.erlang.org should have better uptime than erlang.org, so I would like to have links pointing to it as a first step. That way if we ever move any of the contents from erlang.org to www.erlang.org we will not have to rely on erlang.org to redirect us to the correct place.

Reply:
> So should it discard the two commits that replace /doc/search ?

yes. Feel free to remove the redirect rule from `_redirects` as well.

Reply:
ok. I will do it in a different PR

Reply:
> > But it is redirecting to erlang.org
> 
> [www.erlang.org](http://www.erlang.org) should have better uptime than erlang.org, so I would like to have links pointing to it as a first step. That way if we ever move any of the contents from erlang.org to [www.erlang.org](http://www.erlang.org) we will not have to rely on erlang.org to redirect us to the correct place.

Should this apply to everything, or just downloads?

Reply:
I mean, should this apply to all the first set of rules in _redirects ?

Reply:
Ok. I have left the `_redirects` as the single source of truth, nothing in the site links to https://erlang.org now
I think it reduces the overhead A LOT